### PR TITLE
add some Turkish related translations

### PR DESCRIPTION
**Pull request by behramcelen**
we are using this bundle in our project, i will send pr when we add extra translation.

### CODE REVIEW

File: Resources/translations/time.bs_Latn_BA.xliff [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>diff.ago.year</source>
+                <target>prije 1 godine|%count% godina</target>
```
Comment (delirehberi) [inline Resources/translations/time.bs_Latn_BA.xliff]: You cloud use like this `prije 1 dan|]1,Inf]%count% dana` .

**Comment by weaverryan** [comment]
Hey @behramcelen!

I'd like to merge this, but I want to make sure the correct languages are merged with the correct codes. I'm using Symfony's core as a reference (https://github.com/symfony/symfony/tree/master/src/Symfony/Component/Validator/Resources/translations). There are two problems:

1) `time.hr_HR.xliff` should just be `time.hr.xliff`
2) `time.bs_Latn_BA.xliff` - I don't see Bosnian translated in Symfony's core (or in FOSUserBundle). For that reason, I'm not sure if we should include it.

Thanks!

File: Resources/translations/time.tr.xliff [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                <source>diff.ago.hour</source>
+                <target>1 saat önce|%count% saat önce</target>
+            </trans-unit>
+            <trans-unit id="5">
+                <source>diff.ago.minute</source>
+                <target>1 dakika önce|%count% dakika önce</target>
+            </trans-unit>
+            <trans-unit id="6">
+                <source>diff.ago.second</source>
+                <target>1 saniye önce|%count% dakika önce</target>
```
Comment (taylankasap) [inline Resources/translations/time.tr.xliff]: **dakika** should be changed as **saniye** in this line

**Comment by taylankasap** [comment]
Turkish doesn't use plural forms in this context. Just delete the parts before pipe (|) in the translation file.

**Comment by weaverryan** [comment]
@taylankasap delete *everything* before the `|` for Turkish - e.g. `<target>1 saniye önce|%count% dakika önce</target>` would become `<target>%count% dakika önce</target>`?

**Comment by taylankasap** [comment]
You picked the only wrong translation though. That one already should have been **saniye** instead of **dakika**. But yes, that's what I mean. There are no singular/plural differences in this case.

**Comment by weaverryan** [comment]
@taylankasap ah yes, thank you - I understand well now!

**Comment by weaverryan** [comment]
I've just made those Turkish changes at sha: 47da172845a7e661e5f492da280d73e843d91547

Thanks!

**Pull request merged**